### ISSUE TITLE
[Sybil Report #1] 

### ISSUE THREAD

**Issue by tesloroma** [LayerZero-Labs/sybil-report#4944]
# Wallets:
0xfb233dd9f74a02a444947166e5cbf622037ad24b
0xcb7116d27a4d04d38e8dcdf4e1fadf0bb8f88238
0x4f41a1e056c5eb680baa3e2d4a1b46f8758a060a
0xb5992bc64e2fffc85e7d4f691a280c688702c2cc
0x67844fe8e0c5eec31c73be5a38a1cc584ea8e8a9
0x562c44c24fcdc1d0e864d41a4e53429b5f9bbbc3
0xe1263d6099c1db4a999096d76699b3189f07f1f5
0xb1ff872e72f64d4da3e073782b1bb35506e94050
0x6aafbe590cea24d908d2e48f0bc9a857945d5c24
0x5c16b462d4e78b1b357daf24271d3121f23349a
0x3ee87ea673da600421b95e8e5a14600edabc516d
0x66315fa3b5be254e744291f5df1d3fa49d1ceefe
0x54184ff84f9e6fc0bb95bda940d83bd61636015f
0xa60d57fc09471d88f68db5f7a746641d3227cfe9
0xe9a618dfec37c16e7f2ed9b750a60e2ee25ca4c6
0x4804eb107fe9913e7db165b98cacbee2ac2d2de6
0xf21b77fcfdf0f83f2129cfeda8da50613ae81688
0x3a07c392db4728dba06ad1ba257f3f189dd12da4
0x61b3e832c5de85a3de978d3bdb138b818001068d
0x751180b3c46eac8b2de15c6a1fdd1cd863752dbe
0xbc3aaf164515f5a2ba690c3e6d7c51dbd05fdb78
0x1c93aff61d437822f44811d1edfb489e7c24c0c2
0x0aef6cc9a81dce4f625df90f652add6366d1bf80

# This report stores 23 wallet addresses that were created on the same day and that transferred funds between their wallets in a chain.

# Methodology & Walkthrough:
All of these addresses interacted with the LayerZero project.
A wallet address ( 0x0aef6cc9a81dce4f625df90f652add6366d66d1bf80) was randomly selected, which was further broken into chains. The date of the first transaction of this wallet is 2023-05-31. Going to its transactions in the ether network, we can see that it received funds in the ETH network from another address (0x1c93aff61d437822f44811d1edfb489e7c24c0c2), which was also created on 2023-05-31. Going to it, everything repeats including the last wallet (0xfb233dd9f74a02a444947166e5cbf622037ad24b). All these addresses were created on the same day, funds in the ETH network were transferred directly from one address (starting with 0xfb233dd9f74a02a444947166e5cbf622037ad24b) to another (ending with 0x0aef6cc9a81dce4f625df90f652add6366d1bf80).

If you open all these wallets. They have the same transactions, with the same execution time. In addition, the execution amount is also the same.

In the screenshot (1-5) we can see how the first 5 addresses in the ETH network (0x0aef6cc9a81dce4f625df90f652add6366d66d1bf80,0x1c93aff61d437822f44811d1edfb489e7c24c0c2,0xbc3aaf164515f5a2ba690c3e6d7c51dbd05fdb78, 0x751180b3c46eac8b2de15c6a1fdd1cd863752dbe,0x61b3e832c5de85a3de978d3bdb138b818001068d) have identical transactions including times, protocols, amounts:
![1](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/81b52cf7-73d8-4ba2-a985-98bd640a0a4c)
![2](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/32ae0eec-86ae-420c-892b-ee9e99e72b56)
![3](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/26c9596f-efbb-476a-8ba0-de38bde3c970)
![4](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/93387d3f-bcf0-44f9-94f2-02cc7379aec4)
![5](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/d05cbd97-8b46-4cd4-928c-c2e6701925f9)

In the screenshot (6-10) We observe how 5 addresses in the BNB network (0x3a07c392db4728dba06ad1ba257f3f3f189dd12da4,0xf21b77fcfdf0f83f2129cfeda8da50613ae81688,0x4804eb107fe9913e7db165b98cacbee2ac2d2de6, 0xe9a6a618dfec37c16e7f2ed9b750a60e2ee25ca4c6,0xa60d57fc09471d88f68db5f7a746641d3227cfe9) have identical transactions including times, protocols, amounts:
![6](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/11a9cbb2-6152-4027-a586-35c1e507a2f8)
![7](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/81464891-0ba7-41b7-9cce-d946f5d94c25)
![8](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/ea3645ac-121c-41a9-8e58-96a4deba4b41)
![9](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/1f4667cb-7fad-4e48-a3e3-7bbdbd87b732)
![10](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/dba2e3af-e49a-4615-8f19-094f9b6467ae)

In screenshots (11-23) We observe how 13 addresses in the ZkSync network (0x54184ff84f9e6fc0bb95bda940d83bd61636015f, 0x66315fa3b5be254e744291f5df1d3fa49d1ceefe,0x3ee87ea673da600421b95e8e5a14600edabc516d,0x5c16b462d4e78b1b357daf242751d3121f23349a,0x6aafbe590cea24d908d2e48f0bc9a857945d5c24, 0xb1ff872e72f64d4da3e073782b1bb35506e94050,0xe1263d6099c1db4a999096d76699b3189f07f1f5,0x562c44c24fcdc1d0e864d41a4e53429b5f9bbbc3, 0x67844fe8e0c5eec31c73be5a38a1cc584ea8e8a9,0xb5992bc64e2fffc85e7d4f691a280c688702c2cc,0x4f41a1e056c5eb680baa3e2d4a1b46f8758a060a,0xcb7116d27a4d04d38e8dcdf4e1fadf0bb8f88238
0xfb233dd9f74a02a444947166e5cbf622037ad24b) have identical transactions including times, protocols, amounts.
![11](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/d63b5fca-51cf-4d38-9827-db2286ea222e)
![12](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/c6cc81b6-4c3e-45ef-a6a6-6b02c34f2e71)
![13](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/d1b85ea1-7f0b-488e-a3fa-f98188e5f7a6)
![14](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/9fab90aa-bbcf-4f38-891e-95ffce54b479)
![15](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/5e77cc6f-0141-4cf5-8d7a-5195c9e20806)
![16](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/7768fa04-c78c-47df-8fd2-89b00f3b5c29)
![17](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/2122b5a3-b2c8-4884-a4ff-7633b37e64ce)
![18](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/ccbf839d-5249-457f-a958-3177ebc251fc)
![19](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/c74763be-9ab9-4c11-80a3-c84b5fbb30db)
![20](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/d5ad0266-8fe8-4a86-b125-a55d19ecc81a)
![21](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/0465c4ab-b682-416a-b960-0dc514144035)
![22](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/d1f93ecc-c8a4-4b82-8602-f0becf231f12)
![23](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/475bd33a-3e54-418b-95f8-7c0bce0337d8)

In the screenshots (24-27), the first 4 wallets( 0x0aef6cc9a81dce4f625df90f652add6366d66d1bf80, 0x1c93aff61d437822f44811d1edfb489e7c24c0c2, 0xbc3aaf164515f5a2ba690c3e6d7c51dbd05fdb78,0x751180b3c46eac8b2de15c6a1fdd1cd863752dbe) As stated earlier, transaction identity is present.
![24](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/6ea9265c-8c43-419d-9b96-d9d12c4fedab)
![25](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/0503dffc-a74e-4799-8448-b6614e06cd34)
![26](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/59fdb251-69a0-43a1-aef7-dc4fd3b49ed0)
![27](https://github.com/LayerZero-Labs/sybil-report/assets/26879388/b297e021-938f-45fe-9ea4-c4f239226c87)

To summarize: The list contains 23 wallets. Each of these wallets transferred funds by chain to another. There is the same date of the first transaction (i.e. the wallets were created on the same day). The same networks, the same transactions, time of transactions, transaction amounts says that it is 100% sybil.

This report has been prepared manually, and has been double-checked several times. It is possible to automate the process. We are waiting for feedback!

# My wallet: 0xc762ae74cc89593897851291ca6d84e35191b748
